app/views/shared/dialogs/_dialog_field_drop_down_list.html.haml

```diff
@@ -19,7 +24,12 @@
   - if field.auto_refresh
     :javascript
       dialogFieldRefresh.listenForAutoRefreshMessages("#{field.id}", function() {
-        var selectedValue = $('select[name="#{field.name}"]').val();
+        var field = $('select[name="#{field.name}"]').length;
+        if (field) {
+          var selectedValue = $('select[name="#{field.name}"]').val();
+        } else {
+          var selectedValue = $('select[name="#{field.name}[]"]').val();
+        }
```

Comment:
As far as I know, `.val()` should still work with a multiple select drop down, it should just return an array of the selected values. I saw we have logic to handle the array in `refresh_json_value`, so I'm guessing you ran into issues using `.val` here?